src/types/userOp.ts

```diff
@@ -35,6 +35,7 @@ export interface DecodedError {
 }
 
 export enum ErrorSource {
+    ENTRY_POINT = "ENTRY_POINT",
```

Comment:
invalid UserOpNonce error is thrown from EntryPointContract so should we keep ErrrorSource as `ENTRY_POINT` ? or should it still be set as `SMART_ACCOUNT`

@tomarsachin2271

Reply:
Yeah the error source for this error can be EntryPoint as the actual code checking is done in EntryPoint.
Although there are some errors which are thrown from EP but its because SmartAccount or Paymaster sent that error data to EP... in such cases error source should be SA or Paymaster instead of EP.

But yeah in this case it can be EntryPoint as actual nonce checking is done in EP.